extensions/amp-story/0.1/page-advancement.js

```diff
@@ -279,7 +279,7 @@ class ManualAdvancement extends AdvancementConfig {
 
     // Using `left` as a fallback since Safari returns a ClientRect in some
     // cases.
-    const offsetLeft = elRect.x || elRect.left;
+    const offsetLeft = elRect.x !== null ? elRect.x : elRect.left;
```

Comment:
You probably want abstract equality not strict equality in case `x` returns `undefined`.